manifests/t/Tutanota/Tutanota/3.105.4/Tutanota.Tutanota.installer.yaml

```diff
@@ -11,21 +11,18 @@ InstallModes:
 - interactive
 - silent
 UpgradeBehavior: install
-AppsAndFeaturesEntries:
-- DisplayName: Tutanota Desktop 3.104.5
```

Comment:
Why are the ARP Entries removed rather than updated

Reply:
Generally WinGet does detect DisplayName when it is just `PackageName` + `PackageVersion` so there isn't any need for it per se, but I guess it won't hurt adding it back. Its removal would have made more sense if this package automatically updated itself so we won't have to manually update AppsAndFeatureEntries everytime, but since this package already requires manual PRs, I might as well add this back.